plugin-core/src/main/java/com/salesforce/bazel/eclipse/launch/BazelJavaSourceLookupDirector.java

```diff
@@ -33,17 +33,23 @@
  */
 package com.salesforce.bazel.eclipse.launch;
```

Comment:
With these changes setting breakpoints in source code of jars (dependencies) works